src/main/java/whisper/WhisperException.java

```diff
@@ -0,0 +1,46 @@
+package whisper;
+import java.util.*;
```

Comment:
can import specific classes instead of entire packages!

Reply:
noted on this.